### PR TITLE
Review #601

### PR DESCRIPTION
**Pull request by kozo2**

### CODE REVIEW

File: src/resources/language/_language-ja.yml [hunk 1]
```diff
@@ -69,17 +69,17 @@ environment-remark-title: "注釈"
...
-listing-page-order-by-default: "通常"
-listing-page-order-by-date-asc: "最古"
-listing-page-order-by-date-desc: "最新"
-listing-page-order-by-number-desc: "降順"
-listing-page-order-by-number-asc: "昇順"
+listing-page-order-by-default: "デフォルト"
+listing-page-order-by-date-asc: "日付の昇順"
+listing-page-order-by-date-desc: "日付の降順"
+listing-page-order-by-number-desc: "ページ番号の降順"
+listing-page-order-by-number-asc: "ページ番号の昇順"
```
Comment (eitsupi) [inline src/resources/language/_language-ja.yml:76]: "日付（昇順）", "ページ番号（昇順）" ?

**Review by eitsupi — COMMENTED** [review]
@kozo2 Thanks for the review. However there was no ping and I was not aware of it.

I have a couple of areas that I think are inadequate.

File: src/resources/language/_language-ja.yml [hunk 2]
```diff
@@ -69,17 +69,17 @@ environment-remark-title: "注釈"
...
 listing-page-field-date: "日付"
 listing-page-field-title: "題名"
 listing-page-field-description: "説明"
 listing-page-field-author: "作者"
 listing-page-field-filename: "ファイル名"
-listing-page-field-filemodified: "変更"
+listing-page-field-filemodified: "ファイル変更"
```
[the remark this answers is not in the document]
Comment (eitsupi) [reply src/resources/language/_language-ja.yml:82]: Sorry, it is a date.
https://quarto.org/docs/websites/website-listings.html#listing-fields

I think it is redundant to bother to include "ファイル".

[the remark this answers is not in the document]
Comment (eitsupi) [reply src/resources/language/_language-ja.yml:82]: "更新"?

**Comment by eitsupi** [comment]
I have created a new PullRequest #668.

**Comment by jjallaire** [comment]
Thank! @kozo2 and @eitsupi please let me know when you think this is ready for merge.

**Pull request merged**